<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="anotate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear annotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCSV">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load CSV file</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>1050</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleGraph">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xlabelGraph">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>255</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ylabelGraph">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listX">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>375</y>
      <width>200</width>
      <height>325</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
